testing/unit/tp/node_account/test_transfer_frozen_to_unfrozen.py

```diff
@@ -0,0 +1,221 @@
+from sawtooth_sdk.protobuf.setting_pb2 import Setting
+from sawtooth_sdk.protobuf.transaction_pb2 import (
+    Transaction,
+    TransactionHeader,
+)
+
+from remme.protos.node_account_pb2 import (
+    NodeAccount,
+    NodeAccountMethod,
+    NodeAccountInternalTransferPayload,
+)
+from remme.protos.transaction_pb2 import TransactionPayload
+from remme.settings import ZERO_ADDRESS, SETTINGS_MINIMUM_STAKE
+from remme.settings.helper import _make_settings_key
+from remme.shared.utils import hash512
+from remme.tp.account import AccountHandler
+from remme.tp.node_account import NodeAccountHandler
+from testing.conftest import create_signer
+from testing.mocks.stub import StubContext
+from testing.utils.client import proto_error_msg
+
+RANDOM_NODE_PUBLIC_KEY = '039d6881f0a71d05659e1f40b443684b93c7b7c504ea23ea8949ef5216a2236940'
+
+FROZEN = 600_000
+UNFROZEN = 100_000
+MINIMUM_STAKE = 250_000
+
+ACCOUNT_ADDRESS_FROM = '116829d71fa7e120c60fb392a64fd69de891a60c667d9ea9e5d9d9d617263be6c20202'
+
+ACCOUNT_FROM_PRIVATE_KEY = '1cb15ecfe1b3dc02df0003ac396037f85b98cf9f99b0beae000dc5e9e8b6dab4'
+
+ADDRESS_TO_GET_MINIMUM_STAKE_AMOUNT = _make_settings_key(SETTINGS_MINIMUM_STAKE)
+
+INPUTS = OUTPUTS = [
+    ACCOUNT_ADDRESS_FROM,
+    ADDRESS_TO_GET_MINIMUM_STAKE_AMOUNT,
+]
+
+TRANSACTION_REQUEST_ACCOUNT_HANDLER_PARAMS = {
+    'family_name': NodeAccountHandler().family_name,
+    'family_version': NodeAccountHandler()._family_versions[0],
+}
+
+
+def create_context(account_from_frozen_balance, account_to_unfrozen_balance):
+    """
+    Create stub context with initial data.
+
+    Stub context is an interface around Sawtooth state, consider as database.
+    State is key-value storage that contains address with its data (i.e. account balance).
+
+    References:
+        - https://github.com/Remmeauth/remme-core/blob/dev/testing/mocks/stub.py
+    """
+    node_account_protobuf = NodeAccount()
+
+    # node_account_protobuf.reputational_balance = account_from_frozen_balance + account_to_unfrozen_balance
+    node_account_protobuf.reputation.frozen = account_from_frozen_balance
+    node_account_protobuf.reputation.unfrozen = account_to_unfrozen_balance
+    serialized_account_balance = node_account_protobuf.SerializeToString()
+
+    minimum_stake_setting = Setting()
+    minimum_stake_setting.entries.add(key=SETTINGS_MINIMUM_STAKE, value=str(MINIMUM_STAKE))
+    serialized_minimum_stake_setting = minimum_stake_setting.SerializeToString()
+
+    initial_state = {
+        ACCOUNT_ADDRESS_FROM: serialized_account_balance,
+        ADDRESS_TO_GET_MINIMUM_STAKE_AMOUNT: serialized_minimum_stake_setting,
+    }
+
+    return StubContext(inputs=INPUTS, outputs=OUTPUTS, initial_state=initial_state)
+
+
+def test_transfer_from_frozen_to_unfrozen_low_frozen_balance():
+    """
+    Case: tokens from reputational frozen balance to reputational unfrozen balance with frozen balance < minimum stake.
+    Expect: Frozen balance is lower than the minimum stake.
```

Comment:
Completely store tests code style, please. Looking for [this](https://github.com/Remmeauth/remme-core/blob/master/testing/unit/tp/public_key/test_store.py#L355):
- lower letters after `Case: ` or `Expect: ` only,
- `invalid transaction error is raised with *message* error message` for cases with errors.

Reply:
@dmytrostriletskyi, done.